docs/pt/docs/advanced/settings.md

```diff
@@ -0,0 +1,485 @@
+
+Assim, o atributo `admin_email` seria definido como `"deadpool@example.com"`.
+
+`app_name` seria `"ChimichangApp"`.
+
+E `items_per_user` manteria o valor padrão de `50`.
+
+## Configurações em um módulo separado
+
+Você também pode incluir essas configurações em um arquivo de um módulo separado como visto em [Bigger Applications - Multiple Files](../tutorial/bigger-applications.md){.internal-link target=\_blank}.
+
+Por exemplo, você pode adicionar um arquivo `config.py` com:
+
+```Python
+{!../../../docs_src/settings/app01/config.py!}
+```
+
+E utilizar essa configuração em `main.py`:
+
+```Python hl_lines="3  11-13"
+{!../../../docs_src/settings/app01/main.py!}
+```
+
+!!! dica
+    Você também precisa incluir um arquivo `__init__.py` como visto em [Bigger Applications - Multiple Files](../tutorial/bigger-applications.md){.internal-link target=\_blank}.
+
+## Configurações em uma dependência
+
+Em certas ocasiões, pode ser útil fornecer essas configurações a partir de uma dependência, em vez de definir um objeto global `settings` que é utilizado em toda a aplicação.
+
+Isso é especialmente útil durante os testes, já que é bastante simples sobrescrever uma dependência com suas configurações personalizadas.
+
+### O arquivo de configuração
+
+Baseando-se no exemplo anterior, seu arquivo `config.py` seria parecido com isso:
+
+```Python hl_lines="10"
+{!../../../docs_src/settings/app02/config.py!}
+```
+
+Perceba que dessa vez não criamos uma instância padrão `settings = Settings()`.
+
+### O arquivo principal da aplicação
+
+Agora criamos a dependência que retorna um novo objeto `config.Settings()`.
+
+=== "Python 3.9+"
+
+    ```Python hl_lines="6  12-13"
+    {!> ../../../docs_src/settings/app02_an_py39/main.py!}
+    ```
+
+=== "Python 3.8+"
+
+    ```Python hl_lines="6  12-13"
+    {!> ../../../docs_src/settings/app02_an/main.py!}
+    ```
+
+=== "Python 3.8+ non-Annotated"
+
+    !!! dica
+        Utilize a versão com `Annotated` se possível.
+
+    ```Python hl_lines="5  11-12"
+    {!> ../../../docs_src/settings/app02/main.py!}
+    ```
+
+!!! dica
+    Vamos discutir sobre `@lru_cache` logo mais.
+
+    Por enquanto, você pode considerar `get_settings()` como uma função normal.
+
+E então podemos declarar essas configurações como uma dependência na função de operação da rota e utilizar onde for necessário.
+
+=== "Python 3.9+"
+
+    ```Python hl_lines="17  19-21"
+    {!> ../../../docs_src/settings/app02_an_py39/main.py!}
+    ```
+
+=== "Python 3.8+"
+
+    ```Python hl_lines="17  19-21"
+    {!> ../../../docs_src/settings/app02_an/main.py!}
+    ```
+
+=== "Python 3.8+ non-Annotated"
+
+    !!! dica
+        Utilize a versão com `Annotated` se possível.
+
+    ```Python hl_lines="16  18-20"
+    {!> ../../../docs_src/settings/app02/main.py!}
+    ```
+
+### Configurações e testes
+
+Dessa forma utilizar um objeto settings diferente durante os testes torna-se fácil ao criar uma sobrescrita de dependência em `get_settings`:
```

Comment:
Eu traduziria essa frase como:

> Then it would be very easy to provide a different settings object during testing by creating a dependency override  for get_settings:

Então seria muito fácil fornecer uma configuração diferente durante a execução dos testes sobrescrevendo a dependência de `get_settings`: